data/json/monsterdrops/zombie_soldier.json

```diff
@@ -14,14 +14,14 @@
       },
       {
         "distribution": [
-          { "group": "military_standard_assault_rifles", "contents-item": "shoulder_strap", "prob": 75 },
-          { "group": "military_standard_lmgs", "contents-item": "shoulder_strap", "prob": 10 },
-          { "group": "military_standard_shotguns", "contents-item": "shoulder_strap", "prob": 5 }
+          { "group": "military_standard_assault_rifles", "contents-item": "shoulder_strap", "prob": 75, "damage": [0, 4] },
+          { "group": "military_standard_lmgs", "contents-item": "shoulder_strap", "prob": 10, "damage": [0, 4] },
+          { "group": "military_standard_shotguns", "contents-item": "shoulder_strap", "prob": 5, "damage": [0, 4] }
```

Comment:
Consider also adding fouling.

Reply:
The attribute for fouling is "dirt", right?
Do you suggest any particular value range for it?

Reply:
Added from 0 to 6000 for now, if you think another value range would be better feel free to mention it.

Reply:
I don't have suggestions for a range, and I did not remember the key for fouling (and couldn't check at the time), so changes seem good.